autoload/EasyMotion.vim

```diff
@@ -399,7 +405,11 @@ function! s:Prompt(message) " {{{
     echohl None
 endfunction " }}}
 function! s:Throw(message) "{{{
-    throw 'EasyMotion: ' . a:message
+    if g:EasyMotion_verbose
```

Comment:
You are correct, I just wanted to stop messages like 'EasyMotion: Cancelled'. Turns out there's a dead variable to do the same, I changed it to global verbose.